packages/api-client/src/main/conversation/ConversationMessageTimer.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+interface ConversationMessageTimer {
```

Comment:
Not sure about the naming here. 
Since this is the payload for the update request (not actually used in business code), I would rather call it `ConversationMessageTimerUpdatePayload` 
WDYT?

Reply:
It's a bit too long, I renamed it to `ConversationMessageTimerUpdate`, keeping it in line with the spec.